src/core2/graph.test.js

```diff
@@ -118,11 +127,12 @@ describe("graph", () => {
       };
 
       function expectNeighborsEqual(
-        actual: Iterable<{|+edge: Edge<any>, +ref: NodeReference|}>,
-        expected: {|+edge: Edge<any>, +ref: NodeReference|}[]
+        actualIterator: Iterable<{|+edge: Edge, +ref: NodeReference|}>,
```

Comment:
If you call this an `actualIterator`, maybe make its type `Iterator<>`?

But why the name change at all?